src/components/GameBoard.vue

```diff
@@ -15,12 +15,27 @@ export default {
       disableDeck: false,
       targetWord: "",
       startTime: "",
+
     };
   },
   mounted() {
     this.shuffleCards();
   },
   methods: {
+    startGame() {
+      console.log("Game started!");
+
+      this.targetWord = "Match The Cards";
+      this.shortmessage.innerHTML = `<span>${targetWord}</span>`;
+
+      this.start.className = "started";
```

Comment:
`this.start` isn't a reference to an element, so to dynamically add a class, you can use attribute binding

see: https://vuejs.org/guide/essentials/class-and-style.html